src/test/java/uk/gov/pay/products/validations/PaymentRequestValidatorTest.java

```diff
@@ -60,6 +60,15 @@ public void shouldError_onCreatePayment_ifJsonPayloadHasNonNumeric() {
         assertThat(errors.get().getErrors(), hasItem("Field [price] must be a number"));
     }
 
+    @Test
+    public void shouldError_onCreatePayment_ifJsonPayloadPriceIsZero() {
+        Optional<Errors> errors = requestValidator.validatePriceOverrideRequest(objectMapper.valueToTree(ImmutableMap.of("price", "0")));
```

Comment:
We can just do `Map.of("price, "0")` now we’re on Java 11 (this file is from the Java 8 days). Since this is a small PR, you could change all the ones in the rest of the file.

Reply:
Done.